cernopendata/cold_storage/catalog.py

```diff
@@ -122,6 +122,7 @@ def reindex_entries(self):
                     logger.info("The second time worked!")
                 except Exception as e:
                     logger.error(f"Doing it again did not help :( {e}")
+        db.session.commit()
```

Comment:
What is the reason of committing only to the end? There is a scenario that we might endup with indexed data while db operation crushes. Imagine that the iteration loop successfully indexes all records and the moment it exits the `while` loop the `db.session.commit()` fails...What about moving the commit after the `try/except`?